datamade/how-to
# Reorganize Heroku, Netlify, and AWS docs, as well as domain name management, to deployment/

## Overview

See title. It also adds a link to our legacy application inventory, as well as a short checklist for annual server maintenance.

Handles #156

## Testing Instructions

* View the new markdown files and confirm they look good and the prose makes sense
* Confirm links, particularly links in the contents lists, resolve and images render
